scripts/biomedical/VirtualMetabolicHuman/format_metabolite.py

```diff
@@ -0,0 +1,489 @@
+import sys
+import pandas as pd
+import numpy as np
+from SPARQLWrapper import SPARQLWrapper, JSON
+import datacommons as dc
+from bioservices import *
+
+def isNaN(num):
+    return num != num
+
+def main():
```

Comment:
wow, a single 500 line function. please consider breaking it up into helper functions and adding more comments, as numbered steps perhaps, like:

1. Add dcid column
2. ...
3. ...

Reply:
Added helper functions !

Reply:
Is there more you can extract? Each block under a comment in this function is a candidate for a helper. It will really help with readability and maintainability.